src/autoPlay.js

```diff
@@ -102,12 +102,17 @@ export default function autoPlay(MyComponent) {
 
     handleInterval = () => {
       const {
+        autoplay,
         children,
         direction,
         onChangeIndex,
         slideCount,
       } = this.props;
 
+      if (!autoplay) {
+        clearInterval(this.timer);
```

Comment:
On my `onMouseEnter` event I set an component autoplay property to `false` but the slide transition after the interval is done. And then its stopped.